.eslintrc

```diff
@@ -72,7 +72,7 @@
     "babel/generator-star-spacing": ["error", { before: false, after: true }],
 
     "react/display-name": ["error", { ignoreTranspilerName: true }],
-
+    "react/prefer-stateless-function": ["off"],
```

Comment:
See the Key Takeaways (TL;DR) in the article you linked to. We should always use pure components.